admin_guide/manage_images.adoc

```diff
@@ -0,0 +1,83 @@
+[[admin-guide-manage-images]]
+= Managing Nodes
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+[[manage-images-overview]]
+== Overview
+
+You can manage
+xref:../architecture/core_components/builds_and_image_streams.adoc#image-streams[images]
```

Comment:
There's no good topic about images. Only a few words is in the linked section, that's why.